src/libraries/Microsoft.PowerFx.Repl/Repl.cs

```diff
@@ -55,7 +56,7 @@ public class PowerFxREPL
         public ParserOptions ParserOptions { get; set; } = new ParserOptions() { AllowsSideEffects = true };
 
         // example override, switching to [1], [2] etc.
-        public virtual string Prompt => "\n>> ";
```

Comment:
>"\n>> [](http://example.com/codeflow?start=40&length=5)

Why this change? does this mean new prompt will now not start on a new line?

Reply:
Was creating a mess when rewriting the current line :)